package_repo_resource.md

```diff
@@ -0,0 +1,71 @@
+---
+RFC: unassigned
+Author: Thom May <tmay@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Add a resource to manage package repositories
+
+Provide a core resource to allow a user to add a package repository to
+their system.
+
+## Motivation
+
+    As a cookbook author,
+    I want to write cookbooks that rely only on core resources,
+    so that I can minimise my dependencies.
+
+## Specification
+
+Create a new `package_repo` resource, similar to 
+```
+package_repo "foo" do
+  description "A descriptive string"
+  base_url "https://some/url"
+  gpgkey "https://some/other/url"
```

Comment:
Yeah, but its written as "create a single packge_repo resource" which certainly sounds like one big leaky abstraction that won't work well.   If the RFC was stated as to pull yum_repository, apt_repository and friends into core chef and keep as much of their API consistent as possible so that a package_repo alias made sense I'd be less worried about it...